docs/migration.md

```diff
@@ -1,58 +1,361 @@
 # Migration to Starlite
 
-Migrating **from either Starlette or FastAPI** to Starlite is rather uncomplicated, because the frameworks are for the most
+Migrating **from either Starlette or FastAPI** to Starlite is rather uncomplicated, because the frameworks are for the
```

Comment:
Simple instead of rather uncomplicated?

Reply:
That would be too uncomplicated :P
I'll reword the prose sections when I'm done with the rest.